crates/ere-risczero/Cargo.toml

```diff
@@ -9,14 +9,14 @@ license.workspace = true
 zkvm-interface = { workspace = true }
 anyhow = "1.0"                                               #TODO: remove only needed in tests
 toml = "0.8"
-risc0-zkvm = { version = "^2.2.0", features = ["unstable"] }
+risc0-zkvm = { version = "^2.3.0", features = ["unstable"] }
```

Comment:
Had to update this now since the CI was complaining -- this is craving moving to dockerized compiling. I'll try to do that soon for risczero at least in the upcoming workspace change that I had planned.